tests/fixtures/mbtiles/uncompressed_mvt.sql

```diff
@@ -0,0 +1,22 @@
+ANALYZE sqlite_schema;
+INSERT INTO sqlite_stat1 VALUES('tiles','tile_index','196 28 3 1');
+INSERT INTO sqlite_stat1 VALUES('metadata','name','11 1');
```

Comment:
you don't need stats - its an internal sqlite state that should not be copied. Also we probably don't need `analyze sqlite_schema`